cmd/purge.go

```diff
@@ -0,0 +1,200 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/formancehq/go-libs/v2/logging"
+	"github.com/formancehq/go-libs/v2/temporal"
+	"github.com/formancehq/payments/internal/connectors/engine"
+	"github.com/spf13/cobra"
+	"go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/metric/noop"
+	"go.temporal.io/api/enums/v1"
+	"go.temporal.io/api/workflow/v1"
+	"go.temporal.io/api/workflowservice/v1"
+	"go.temporal.io/sdk/client"
+	"go.uber.org/fx"
+)
+
+func newPurge() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "purge",
+		Short:        "Launch purge command to clean temporal",
+		SilenceUsage: true,
+		RunE:         runPurge(),
+	}
+
+	temporal.AddFlags(cmd.Flags())
+
+	return cmd
+}
+
+func runPurge() func(cmd *cobra.Command, args []string) error {
+	return func(cmd *cobra.Command, args []string) error {
+		setLogger()
+
+		return purgeOptions(cmd)
+	}
+}
+
+func purgeOptions(cmd *cobra.Command) error {
+	temporalNamespace, _ := cmd.Flags().GetString(temporal.TemporalNamespaceFlag)
+	stack, _ := cmd.Flags().GetString(StackFlag)
+	logger := logging.NewDefaultLogger(cmd.OutOrStdout(), true, true, false)
```

Comment:
Didn't @gfyrag want to get rid of the logging package in the near future? Maybe we should refrain from requiring it in new code?

Reply:
I think we will do a big pass on all the payments code when it's done in go-libs, what do you think ?